asserts/account_key.go

```diff
@@ -43,49 +43,41 @@ func (ak *AccountKey) Since() time.Time {
 	return ak.since
 }
 
-// Until returns the time when the account key stops being valid.
+// Until returns the time when the account key stops being valid. A zero time means the key is valid forever.
 func (ak *AccountKey) Until() time.Time {
 	return ak.until
 }
 
-// PublicKeyID returns the key id (as used to match signatures to signing keys) for the account key.
-func (ak *AccountKey) PublicKeyID() string {
-	return ak.pubKey.ID()
-}
-
-// PublicKeyFingerprint returns the fingerprint of the account key.
-func (ak *AccountKey) PublicKeyFingerprint() string {
-	return ak.pubKey.Fingerprint()
+// PublicKeySHA3_384 returns the key SHAS3-384 hash used for lookup of the account key.
+func (ak *AccountKey) PublicKeySHA3_384() string {
+	return ak.pubKey.SHA3_384()
```

Comment:
Can we please keep both of those as `AccountKey.PublicKeyID` and `pubKey.ID`? This is still much nicer to write, read, and understand.